cf/data/data.py

```diff
@@ -6366,42 +6366,22 @@ def mask(self):
         (12, 73, 96)
         >>> m = d.mask
         >>> m.dtype
-        dtype('bool')
+        bool
```

Comment:
```suggestion
        dtype('bool')
```
since:
```python
>>> d = cf.Data([1, 0, 0], dtype=bool)
>>> d.dtype
dtype('bool')

Reply:
Ah yes, apologies, this was left over from my initial change-of-behaviour idea which was rightly scrapped...